CHANGELOG.md

```diff
@@ -268,6 +268,7 @@ the stability guarantees we provide for pre-releases.
 
 ### IMPROVEMENTS
 
+- crypto/merkle.innerHash is much more efficient ([#1921](https://github.com/cometbft/cometbft/pull/1921))
```

Comment:
```suggestion
```

Please see https://github.com/cometbft/cometbft/blob/main/CONTRIBUTING.md#changelog

Reply:
Done, than you @melekes!